types/node/assert.d.ts

```diff
@@ -46,7 +46,7 @@ declare module "assert" {
                 operator?: string | undefined;
                 /** If provided, the generated stack trace omits frames before this function. */
                 // eslint-disable-next-line @typescript-eslint/ban-types
-                stackStartFn?: Function | undefined;
+                /** @deferred */ stackStartFn?: Function | undefined;
```

Comment:
This one definitely doesn't work; this tag is not on a parameter. Not sure how many of these are also incorrect, but if you're using a script for this it may need to be tweaked.

Reply:
this one was manual and just I missed the `options?: {`, although I used find/replace for the nearly all the rest of them which is *definitely* not going to notice that. I'll try compiling with the branch to see what complaints I get.